SonarQube.Client/Models/ServerExclusions.cs

```diff
@@ -57,5 +57,23 @@ public override string ToString()
                    "\n    Exclusions: " + string.Join(",", Exclusions) +
                    "\n    Global Exclusions: " + string.Join(",", GlobalExclusions);
         }
+
+        public override bool Equals(object obj)
+        {
+            return Equals((ServerExclusions)obj);
+        }
+
+        public bool Equals(ServerExclusions other)
+        {
+            if (ReferenceEquals(null, other)) return false;
+            if (ReferenceEquals(this, other)) return true;
```

Comment:
I think we always use curly-brackets, even for single-line statements as simple as this one.